### ISSUE TITLE
Example step definitions are not escaped

### ISSUE THREAD

**Issue by robinelvin** [bencompton/jest-cucumber#37]
Automatically generated code for unimplemented scenarios sometimes generates invalid Javascript.
For example, with a step such as

```gherkin
Given the user's email is blank
```

The example code generates

```javascript
given('the user's email is blank', () => {
    pending();
});
```

when it should be

```javascript
given('the user\'s email is blank', () => {
    pending();
});
```

**Comment by bencompton** [comment]
Good catch--definitely should escape the single quotes.

**Comment by bencompton** [comment]
This is fixed in version 2.0.2 on NPM, which is currently a beta version (tagged @next).
